mozci/util/taskcluster.py

```diff
@@ -81,6 +83,55 @@ def find_task_id(index_path, use_proxy=False, root_url=PRODUCTION_TASKCLUSTER_RO
     return response.json()["taskId"]
 
 
+def insert_task(
```

Comment:
You could directly create `index_current_task` here:
- do not pass `task_id` and use directly `os.environ["TASK_ID"]` or raising an error when it's not available
- `use_proxy` seems unused ?

Reply:
Sure, I implemented it like this so that the function is easily reusable, but I can refactor it.